lib/ReactViewModels/ViewState.ts

```diff
@@ -562,16 +565,11 @@ export default class ViewState {
           return Result.error(`Could not view catalog member ${getName(item)}`);
         }
       }
-      const theItem: BaseModel =
-        ReferenceMixin.isMixedInto(item) && item.target ? item.target : item;
```

Comment:
Doubt: why don't we need to resolve the target any more? Is it because we are already recursively calling `viewCatalogMember` when item is a reference?

Reply:
Yep, it should be handled by

https://github.com/TerriaJS/terriajs/blob/e6615c39b89eb209330008f0b824ab15da106d26/lib/ReactViewModels/ViewState.ts#L560-L567